msm/src/mvlookup.rs

```diff
@@ -361,8 +375,9 @@ pub mod prover {
                 for i in 0..domain.d1.size as usize {
                     // phi(1) = 0
                     evals.push(acc);
-                    // FIXME: only one lookup for now.
-                    acc += lookup_terms_evals_d1[i]
+                    for lte in lookup_terms_evals_d1.iter() {
+                        acc += lte[i]
```

Comment:
I'm not exactly sure what's the intention here. I'm thinking you want to kind of create a joint accumulator for several lookup arguments in parallel, but then they are becoming just one bigger argument. Semantically that's not what I'd expect tbh, I kind of expected them to be independent. If we wanted to support more queries, one could already add more `f_i` into already existing argument. If we wanted more values to look up, we could instead support multiple `t_i` instead. With the design that I see one basically needs to create an additional lookup argument if one has too much elements to look up, instead of extending the previous argument. And so one must optimise number of `f_i` for `t` in each batch... instead of just appending `t_i`s and `f_i`s to one argument. Am I missing smth?